src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTestBase.cs

```diff
@@ -32,13 +32,21 @@ public HttpClientHandlerTestBase(ITestOutputHelper output)
         protected virtual HttpClient CreateHttpClient() => CreateHttpClient(CreateHttpClientHandler());
 
         protected HttpClient CreateHttpClient(HttpMessageHandler handler) =>
-            new HttpClient(handler) { DefaultRequestVersion = UseVersion };
+            new HttpClient(handler) {
+#if !NETFRAMEWORK
+                DefaultRequestVersion = UseVersion
+#endif
+            };
 
         protected static HttpClient CreateHttpClient(string useVersionString) =>
             CreateHttpClient(CreateHttpClientHandler(useVersionString), useVersionString);
 
         protected static HttpClient CreateHttpClient(HttpMessageHandler handler, string useVersionString) =>
-            new HttpClient(handler) { DefaultRequestVersion = Version.Parse(useVersionString) };
+            new HttpClient(handler) {
+#if !NETFRAMEWORK
+                DefaultRequestVersion = Version.Parse(useVersionString)
```

Comment:
`useVersionString` isn't used here, is that intended? if so, can you assert the value you're expecting it to be?

Reply:
Framework doesn't have DefaultRequestVersion property. There is nothing to assert at because all values are valid, but on Framework they are applied at a [different place](https://github.com/dotnet/runtime/pull/33158/files/03d72f7411c47c6fcc0a8842c51755cc43cf9e3f#diff-abf8cb32c7cb51a4ac59bcad5c021512R185)